packages/flutter/lib/src/widgets/icon_theme.dart

```diff
@@ -49,9 +49,18 @@ class IconTheme extends InheritedTheme {
   final IconThemeData data;
 
   /// The data from the closest instance of this class that encloses the given
-  /// context.
+  /// context, if any.
   ///
-  /// Defaults to the current [ThemeData.iconTheme].
+  /// If there is no ambient icon theme, defaults to [IconThemeData.fallback].
```

Comment:
I believe @HansMuller had plans to eventually stop using `fallback` for this kind of thing. That said we should document how it currently works, so this is a nice clarification.

Reply:
I think I was referring to ThemeData.fallback(). This looks like a helpful clarification.